sql/moz-fx-data-shared-prod/telemetry/attributed_windows_desktop_funnel_installs/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
```

Comment:
Do we expect to provide a single unified view on top of installs, new profiles, and sessions? If that's what we want end users to reference, then it might not make sense to publish views for each of these individually.

Reply:
This makes sense to me, I'll take these out and actually create the unified view.